pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -597,6 +596,41 @@ public static String splitPart(String input, String delimiter, int index) {
     }
   }
 
+  /**
+   * TODO: Revisit if index should be one-based (both Presto and Postgres use one-based index, which starts with 1)
```

Comment:
nit: we can remove this TODO. I see it's still there in `splitPart`, but once we ship this we anyways can't change this without making the function b/w incompatible. Same for the next function